docs/analysis-tools/variant-distance-matrix.md

```diff
@@ -0,0 +1,40 @@
+**Variant Distance Matrix**
+===============
+About the Variant Distance Matrix Tool
+--------------------------
+The Variant Distance Matrix tool calculates the disimilarity between variant sets in the database and stores the results back into the database. It only calculates the distances between variant sets of the same species and includes sanity checks to ensure that duplicate or improper calculations do not occur. This allows for other tools, most notably the SimilarVariantSet tool, to quickly determine relationships between variant sets. 
+
+The Variant Distance Matrix tool makes one assumption when calculates the distance between variant sets: the genome is static within the same species. In other words, the variant distance matrix tool assumes that any two variant sets for which the distance is calculated for have the same reference genome. This allows the tool to assume that the distance between two variant sets is only a function of the differences between the variants in the variant sets and not any other outside variables. Since both variant sets have the same static genome, the distance is calculated using the following formula:
+
+![variant distance equation](images/variant_distance_equation.png)
+
+Using the Variant Distance Matrix Tool
+--------------------------
+This tool mainly functions as a helper tool to the VariantBatchUpload and SimilarVariantSet tools and is unable to be called from the website. Upon the upload of a VariantSet, multiple Variant Distance Matrix instances are forked off to calculate the distances between the new VariantSet and all existing variant sets in the database. The SimilarVariantSet uses this tool when any two uploaded variant sets do not already have an existing distance in the database. The SimilarVariantSet then forks of instances of the Variant Distance Matrix tool in order to calculate these distances.
```

Comment:
I'm not sure what this means: "The SimilarVariantSet then forks of instances of" in the last sentance